contribs/gmf/src/directives/displayquerygrid.js

```diff
@@ -90,7 +90,7 @@ gmf.displayquerygridComponent = {
     'removeEmptyColumnsFn': '&?gmfDisplayquerygridRemoveemptycolumns',
     'maxResultsFn': '&?gmfDisplayquerygridMaxresults',
     'maxRecenterZoomFn': '&?gmfDisplayquerygridMaxrecenterzoom',
-    'mergeTabsFn': '&?gmfDisplayquerygridMergetabs'
+    'mergeTabs': '=?gmfDisplayquerygridMergetabs'
```

Comment:
`<` is better than `=` in this case. Because the change can only be from the main controller to this controller, never in the  other way. So that's useless to watch the change in this controller.